interface-definitions/include/dhcp/ddns-settings.xml.i

```diff
@@ -0,0 +1,97 @@
+<!-- include start from dhcp/ddns-settings.xml.i -->
+<leafNode name="send-updates">
+<properties>
+    <help>Send updates for this scope</help>
+    <valueless/>
+</properties>
+</leafNode>
+<leafNode name="override-client-update">
```

Comment:
Maybe call it `force-client-update`? It's usually `force` in our commands in other places.